wildfly-elytron-audit/src/main/java/org/wildfly/security/audit/_private/ElytronMessages.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.security.audit._private;
```

Comment:
I'm not sure how you guys feel but I don't think it makes sense to have a `_private` subpackage when there's only one package in the module.  I would save a few bytes and just make the `ElytronMessages` package-private in this case.

Reply:
Have added a comment to our todo list.